Packs/Cisco_OpenDNS/Integrations/CiscoOpenDNS/CiscoOpenDNS.yml

```
@@ -0,0 +1,67 @@
+category: Network Security
+commonfields:
+  id: Cisco_OpenDNS
+  version: -1
+configuration:
+- defaultvalue: https://s-platform.api.opendns.com/1.0/
+  display: Server URL (e.g. https://example.net)
+  name: url
+  required: true
+  type: 0
+- display: API Key
+  name: APIKey
+  required: true
+  type: 0
```

Please keep sensitive parameters as encrypted (type 4). For more information see [here](https://xsoar.pan.dev/docs/integrations/yaml-file#common-fields)
```suggestion
  type: 4
```